docs/source/pillars/information_governance.md

```diff
@@ -156,64 +297,202 @@ _The ability of the TRE organisation to manage projects effectively._
+  - Mandatory
+* - The data asset owner must have control of who has access to which assets.
+  - Either through policy and process or ideally direct control access to data should be controlled by the person who owns a data asset or their delegates.
+  - Mandatory
+```
+
+**Authentication Application:** A software system that verifies and validates the identities of users or entities accessing a system through multifactor authentication.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-authentication-application
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must have robust and secure applications in place to authenticate users (and services) within the TRE.
+  - The number of authentication applications should be kept to a minimum with common controls and standards applied across all such as MFA, password complexity etc.
+  - Mandatory
+```
+
+**User Identity Attributes:** Characteristics or attributes associated with a user's identity, such as username, email address, role, permissions, or affiliations.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-user-identity-attributes
+
+* - Statement
+  - Guidance
+  - Importance
+* - You must give each user of the TRE a unique logon with changes to any records strictly controlled.
+  - The unique identifier and all associated records for a user should be traceable across the entire TRE.
+    This will include training records, affiliations, contract agreements and ethics approvals where required.
+  - Mandatory
+```
+
+## Training Delivery and Management
+
+Ability to deliver, track and maintain adequate training levels to ensure competence of all people within the TRE organisation.
+
+**Curriculum Creation and Management:** The process of designing, developing, and managing educational curricula, courses through training needs analysis for required competency.
+
+```{list-table}
+:header-rows: 1
+:name: tab-ig-curriculum-creation-management
 
 * - Statement
   - Guidance
   - Importance
 * - You must have relevant training for all roles within the TRE organisation, and the ability to deliver this training.
   - This may include, for instance, cyber security training, GDPR training, and higher level training for system operators.
+    Specific training should be designed for roles such as Senior Information Risk Owner, Data Asset owner.
+    Identification of these specialities should be done through a systematic training needs analysis.
+    Specific training may also be required based on the data or data provider such as GCP.
   - Mandatory
-* - You must ensure that users are aware which actions are and are not permitted in your TRE.
-  - This is likely to depend on the security controls in place in your TRE.
+* - You should have a training platform capable of delivering online training in a variety of formats.
+  - This should include competency testing and more simple recording of actions such as read and understood.
+  - Recommended
```

Comment:
```suggestion
* - You could have a training platform capable of delivering online training in a variety of formats.
  - This could include competency testing and more simple recording of actions such as read and understood.
  - Optional
```